packages/TCSLib/TCSLib.0.5/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "A multi-purpose library for OCaml"
+description: "A multi-purpose library for OCaml."
+maintainer: ["Oliver Friedmann" "Martin Lange"]
+authors: ["Oliver Friedmann" "Martin Lange"]
+license: "BSD-3-clause"
+homepage: "https://github.com/tcsprojects/tcslib"
+bug-reports: "https://github.com/tcsprojects/tcslib/issues"
+depends: [
+  "ocaml" {>= "4.8"}
+  "dune" {>= "3.10"}
+  "num"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
```

Comment:
I think this should fix the build issue:

```suggestion
    "tcslib"
```

The trick here, I think, is just that the name of dune library does not match the opam package name, so we cannot use opam's `name` variable here.